### PR TITLE
added BindKey, allows key bindings for clicking a link or submitting …

### PR DESCRIPTION
**Pull request by edraut**
…a form

more behaviors can be added in the future if needed

### CODE REVIEW

**Comment by edraut** [comment]
@gomayonqui I'm trying to fix circle for hooch. I sent them an email, hopefully they'll have a fix for us.

File: app/assets/javascripts/hooch.js [hunk 1]
```diff
@@ -1088,6 +1088,114 @@ var initHooch = function(){
       replaceWith: function($jq_obj){
         this.$sort_element.replaceWith($jq_obj)
       }
+    }),
+    key_code_map: { // borrowed from jresig: https://github.com/jeresig/jquery.hotkeys
+          "backspace": 8,
+          "tab": 9,
+          "return": 10,
+          "return": 13,
```
Comment (darkside) [inline app/assets/javascripts/hooch.js:1096]: @edraut we have 2 "return" for 2 different characters?

File: lib/hooch/railtie.rb [hunk 2]
```diff
@@ -2,7 +2,6 @@
 module Hooch
   class Railtie < Rails::Railtie
     initializer "hooch.hooch_helper" do
-      puts "initializing hooch helpers"
```
Comment (darkside) [inline lib/hooch/railtie.rb:5]: :+1: :smiley_cat:

File: app/assets/javascripts/hooch.js (hunk 1 above)
Comment (edraut) [inline app/assets/javascripts/hooch.js:1096]: I guess I need an array. bah. I'll make a todo for it, I seriously doubt we'll ever use this again.

**Pull request merged**